detector/vuls2/vendor.go

```diff
@@ -115,11 +134,107 @@ func ignoreCriterion(e ecosystemTypes.Ecosystem, cn criterionTypes.FilteredCrite
 		default:
 			return false
 		}
+	case ecosystemTypes.EcosystemTypeUbuntu:
+		if func() bool {
+			lhs, _, _ := strings.Cut(string(tag), "_")
+			lhs, rhs, ok := strings.Cut(lhs, "/")
+			if !ok {
+				return false
+			}
+			services := []string{"esm", "esm-infra", "esm-apps", "esm-infra-legacy", "esm-apps-legacy", "ros-esm"}
+			if !slices.Contains(services, lhs) && !slices.Contains(services, rhs) {
```

Comment:
Can one of services be rhs? If tag is composed like "esm-apps/jammy_low", only testing lhs suffices?